truffenyi-commune-quest.lic

```diff
@@ -0,0 +1,69 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#truffenyi-commune-quest
+=end
+
+custom_require.call(%w(common))
+
+echo "This script does the actions for the truffenyi commune Cleric quest. It should be started after drinking the vial two times."
+fput("stow #{checkright || checkleft}") if DRC.left_hand || DRC.right_hand
```

Comment:
What do you think about [creating an instance of EquipmentManager](https://github.com/rpherbig/dr-scripts/blob/1c8306610f09bbfa783d980f41b582c1f21f2b17/favor.lic#L12) and calling the `stow_hands` method on it?

Reply:
done